scripts/hello_world_mckenzieja.js

```diff
@@ -0,0 +1,5 @@
+function sup(){
```

Comment:
Where are you comments with the requirements??? See issue #1292

Reply:
Updated!

Reply:
Great! Looks good now!